### PR TITLE
use bookinfo 0.2.3 images due to recent changes in code

### PR DESCRIPTION
**Pull request by rshriram**
```release-note
NONE
```

### CODE REVIEW

File: tests/e2e/tests/bookinfo/demo_test.go [hunk 1]
```diff
@@ -412,7 +412,7 @@ func TestDbRouting(t *testing.T) {
 
 	respExpr := "glyphicon-star" // not great test for v2 or v3 being alive
 
-	_, err = checkHTTPResponse(u1, tc.gateway, respExpr, 11)
+	_, err = checkHTTPResponse(u1, tc.gateway, respExpr, 10)
```
Comment (douglas-reid) [inline tests/e2e/tests/bookinfo/demo_test.go]: we really need another means of checking this. I suggest killing this test altogether. we have a test for metrics to ratings-v2 in mixer_test.go now. we can augment that to ensure we are using ratings-v2 and kill this test.  thoughts?

Comment (rshriram) [reply tests/e2e/tests/bookinfo/demo_test.go]: I don't mind killing it. @ldemailly ??

Comment (ldemailly) [reply tests/e2e/tests/bookinfo/demo_test.go]: this test in theory tests the addition of the db (switch from not using to using it) but I suppose we can have it "always on"
I actually would like to have a test to ensure we don't loose mysql either (but it's a bit hard to test that from the product page)

I missed the v2 test in mixer, are you sure ? also this one is a different setup, it doesn't hurt to have 2 tests for the feature

let's keep it and I'll extend it to updating the db maybe or using lower / different number of stars

File: tests/e2e/tests/bookinfo/demo_test.go [hunk 2]
```diff
@@ -400,6 +400,8 @@ func setTestConfig() error {
 	return nil
 }
 
+/** Disabled because mixer_test has a better DB test. Keeping this
```
Comment (ldemailly) [inline tests/e2e/tests/bookinfo/demo_test.go]: let's not disable it - it doesn't take long, it doesn't hurt to have 2 tests, and it was the only smoking gun of some auth issue

**Comment by ldemailly** [comment]
thanks a lot for updating the images by the way (somehow I thought they were fine already)

but let's not remove tests

**Comment by rshriram** [comment]
done. please approve. bot isnt testing

File: samples/apps/bookinfo/src/mongodb/ratings_data.json [hunk 3]
```diff
@@ -1,2 +1,2 @@
+{rating: 6}
 {rating: 5}
-{rating: 4}
```
Comment (ldemailly) [inline samples/apps/bookinfo/src/mongodb/ratings_data.json]: maybe 1 and 4 for a 5 total to make it different from the hardcoded one (I forgot what the hardcoded values are)

Comment (rshriram) [reply samples/apps/bookinfo/src/mongodb/ratings_data.json]: Oh shoot. This change should not be there. It should be 5, 4

File: tests/e2e/tests/bookinfo/demo_test.go (hunk 1 above)
Comment (ldemailly) [reply tests/e2e/tests/bookinfo/demo_test.go]: I guess the images must not match the source if this 11->10 is changed but not the values
no big deal I guess we'll fix it later

**Pull request merged**